autoload/LanguageClient.vim

```diff
@@ -548,6 +548,16 @@ function! LanguageClient#textDocument_rangeFormatting(...) abort
     return LanguageClient#Call('textDocument/rangeFormatting', l:params, l:callback)
 endfunction
 
+function! LanguageClient#completionItem_resolve(completion_item, ...)
+    let l:callback = get(a:000, 0, v:null)
```

Comment:
Just for keeping consistent. First parameter should be general parameters for overriding, second parameter is for callback.

Can you please change this to `get(a:000, 1, v:null)`?